appinventor/components/src/com/google/appinventor/components/runtime/CloudDB.java

```diff
@@ -1111,7 +1111,8 @@ public void TagList(List<String> value) {
    * @param tag the tag that has changed.
    * @param value the new value of the tag.
    */
-  @SimpleEvent
+  @SimpleEvent(description = "Indicates that the data in the CloudDB project has changed. Launches an event with the " +
+      "`tag` that has been updated and the `value` it now has.")
```

Comment:
This wording might lead people think that this event launches another event.

I'd recommend: "Event indicating that CloudDB data has changed for the given tag and value."